app/views/users/show.html.erb

```diff
@@ -24,9 +24,12 @@
       Likes: <%= post.likes_counter %></p>
     </div>
   <% end %>
+  <div class="add">
   <button id="show-more-btn">
     <%= link_to 'Show More', user_posts_path(@user) %>
   </button>
+  <button id="show-more-btn"><%= link_to "Create New Post", new_user_post_path(@user) %></button>
+  </div>
 <% else %>
   <p>No recent posts.</p>
```

Comment:
- [ ] [ OPTONAL]
The `Create New Post` button doesn't show on the User detail page when there are no created yet. It will be ideal if this button is added, even when there's no post created yet, to enable the user easily create a post.